### PR TITLE
Create a single PDS + PMD Dockerfile and Docker-Compose file #1353

### LINKED ISSUE #1353: Single PDS + PMD
Create a single PDS + PMD Dockerfile and Docker-Compose file. This is a sub-issue of #764.

### PR DESCRIPTION
**Pull request by zigfridus**
Create a single PDS + PMD Dockerfile and Docker-Compose file.
Closes #1353

### CODE REVIEW

**Review by Jeeppler — CHANGES_REQUESTED** [review]
First of all good work. I was able to run PMD+PDS in my environment, with just a minor issue. Please see the comment for more information.

File: sechub-pds-solutions/pmd/docker/pmd.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+# SPDX-License-Identifier: MIT
+
+export PATH="$TOOL_FOLDER/pmd/bin:$PATH"
+run.sh pmd --rulesets category/java/security.xml,category/jsp/security.xml  --format sarif --dir "$PDS_JOB_SOURCECODE_UNZIPPED_FOLDER" --report-file "$PDS_JOB_RESULT_FILE"
```
[the remark this answers is not in the document]
Comment (zigfridus) [reply sechub-pds-solutions/pmd/docker/pmd.sh:5]: Hello @Jeeppler 
I would like to clarify on scan with all PMD rules. Do you mean all rules including other languages (Ecmascript, HTML, etc) ?
Thanks

**Comment by zigfridus** [comment]
@Jeeppler thanks for your help. ✌
I'm going to continue on #1358

**Comment by de-jcup** [comment]
@zigfridus : Fist of all: Hello and thank you for your contributions :-) 

Inside the PR there is a little issue I want to mention:

### Commit messages
**tl;dr:** please read https://github.com/mercedes-benz/sechub/wiki/Collaboration#commit-messages and use the wanted style in future commits

#### Details
While merging my current branch I have seen a commit from this PR which has very big commit message which  is not like our standard ones:

https://github.com/mercedes-benz/sechub/pull/1355/commits/f41bbad22d78898ce86b059001187590e066da6a
```
add empty mocks folder with dummy file;

- remove -qq option from apt-get;
- add ruleset files with rules for PMD;
- add ALL and SECURITY options for PMD scanning. SECURITY enabled by default;
- upgrade PMD version from 6.45.0 to 6.46.0;
Closes https://github.com/mercedes-benz/sechub/issues/1353
```

Git at command line (Linux) is not able to recognize the headline (second empty line) correctly (maybe the semicolon...)  and does always output the complete commit in history:
```
 * | | |       f41bbad22 2022-05-31 (Valentyn Grygoriev)   - add empty mocks folder with dummy file; - remove -qq option from apt-get; - add ruleset files with rules for PMD; - add ALL and SECURITY options for PMD scanning. SECURITY enabled by default; - upgrade PMD version from 6.45.0 to 6.46.0; Closes #1353

```
I often use the command line GIT tooling and be used to short messages.
I described at https://github.com/mercedes-benz/sechub/wiki/Collaboration#commit-messages how commit messages shall look like in SecHub. Please do me a favour and use this styling in future.  

Best wishes
Albert

**Comment by zigfridus** [comment]
@de-jcup 
Sorry, maybe I missed that explanation about committing messages. I'll use those styling in the future commits.
